src/snapred/backend/recipe/algorithm/ReductionAlgorithm.py

```diff
@@ -278,8 +279,17 @@ def PyExec(self):
         vanadiumFilePath = reductionIngredients.reductionState.stateConfig.vanadiumFilePath
         diffCalPath = reductionIngredients.reductionState.stateConfig.diffractionCalibrant.diffCalPath
 
-        raw_data = self.loadEventNexus(Filename=rawDataPath, OutputWorkspace="raw_data")
-        vanadium = self.loadNexus(Filename=vanadiumFilePath, OutputWorkspace="vanadium")
+        raw_data = self.mantidSnapper.loadEventNexus(
```

Comment:
`raw_data = self.mantidSnapper.LoadEventNexus(`
You need to use the name of the algo in mantid, which all start with uppercase

Reply:
Done.